thoth/storages/sync.py

```diff
@@ -548,6 +550,68 @@ def sync_inspection_documents(
 
     return processed, synced, skipped, failed
 
+def sync_security_indicators_documents(
+    document_ids: Optional[List[str]] = None,
+    force: bool = False,
+    graceful: bool = False,
+    graph: Optional[GraphDatabase] = None,
+    is_local: bool = False,
+) -> tuple:
+    """Sync security indicators results into graph."""
+    if is_local and not document_ids:
+        raise ValueError(
+            "Cannot sync documents from local directory without explicitly specifying a list of documents to be synced"
+        )
+
+    if not graph:
+        graph = GraphDatabase()
+        graph.connect()
+
+    if not is_local:
+        security_store = SecurityIndicatorsResultsStore()
+        security_store.connect()
+
+    processed, synced, skipped, failed = 0, 0, 0, 0
+    for security_indicator_key in document_ids or security_store.get_listing():
+        security_indicator_id = security_indicator_key
+
+        if not is_local:
+            security_indicator_id = security_indicator_key.split("/")[0]
+
+        if is_local or "aggregated" in security_indicator_key:
```

Comment:
I don't think `"aggregated" in security_indicator_key` is required or any if statement for that matter.

Reply:
> I don't think `"aggregated" in security_indicator_key` is required or any if statement for that matter.

Actually it is, as ceph returns the complete key/path and under the same security-indicators- three files are found it will iterate three times, but we consider only `aggregated`